tensorboard/webapp/tbdev_upload/tbdev_upload_dialog_component.css

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+:host > div,
+:host > p {
+  margin: 16px 0;
+}
+
+:host > :first-child {
+  margin-top: 0;
+}
+
+:host > :last-child {
+  margin-bottom: 0;
+}
+
+h3 {
+  font-size: 14px;
+  line-height: 20px;
```

Comment:
20px -> 1.5 (so it is relative to the font-size)

Reply:
Done. (and elsewhere)